internal/server/iigointernal/orchestration.go

```diff
@@ -6,13 +6,16 @@ import (
 	"github.com/SOMAS2020/SOMAS2020/internal/common/roles"
 	"github.com/SOMAS2020/SOMAS2020/internal/common/rules"
 	"github.com/SOMAS2020/SOMAS2020/internal/common/shared"
+	"github.com/SOMAS2020/SOMAS2020/internal/common/voting"
 	"github.com/pkg/errors"
 )
 
+const serviceCharge = shared.Resources(10)
```

Comment:
what?

Reply:
Included this as a quick fix to avoid subtracting 10 from the budget every time there was a deduction. This is going to be overhauled post-MVP as there won't be a flat rate for actions.